fee_calculator/apps/calculator/fixtures/price.json

```diff
@@ -34808,7 +34808,7 @@
     "advocate_type": "QC",
     "offence_class": null,
     "fee_type": 12,
-    "unit": "HALFDAY",
```

Comment:
tallies with these [regulations](http://www.legislation.gov.uk/uksi/2013/435/schedule/1/paragraph/24/made) for "committal for sentence"

<img width="871" alt="screen shot 2018-07-06 at 12 17 23" src="https://user-images.githubusercontent.com/7016425/42376316-a6429c1c-8116-11e8-8cd2-bd786baad713.png">